pandas/tests/extension/base/casting.py

```diff
@@ -36,6 +36,11 @@ def test_astype_str(self, data):
         expected = pd.Series(data[:5].astype(str))
         self.assert_series_equal(result, expected)
 
+    def test_astype_string(self, data):
+        result = pd.Series(data[:5]).astype("string")
+        expected = pd.Series(data[:5].astype("string"))
```

Comment:
Can you create the expected without relying on the EA method (eg doing the `astype(str)` workaround) ?

Reply:
Ok.